home/import_content_pages.py

```diff
@@ -69,7 +77,10 @@ def process_rows(self, rows: list["ContentRow"]) -> None:
     def save_pages(self) -> None:
         for i, page in enumerate(self.shadow_pages.values()):
             if page.parent:
-                parent = Page.objects.get(title=page.parent, locale=self.locale)
+                # TODO: We should need to use something unique for `parent`
+                parent = Page.objects.filter(
+                    title=page.parent, locale=self.locale
+                ).first()
```

Comment:
Where did this assumption break? It's never actually been valid, but having the import fail is probably better than silently reparenting pages.

Reply:
This broke when using data on my local system, which is from an export of a production system. I think it is better to fail here, so maybe I'll change it back, although that will break backwards compatibility for old exports.